### ISSUE TITLE
twisted.vfs.test.test_vfs.OSVFSTest uselessly deletes its working directory in tearDown

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3320]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)| @therve reported|
|-|-|
|Trac ID|trac#3320|
|Type|defect|
|Created|2008-06-28 16:22:39Z|

Windows doesn't like it, sometimes.

Attachments:

* [vfs.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/fc8/fc839ea7ccd35a6e07a51188e6af9c05e5b3194f/553ba94dd6ea2823761e965d6c705c601e5a19c9.diff) (785 bytes) - added by therve on 2008-06-28 16:23:16Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__3320 3320
type__defect defect
reporter__therve therve
priority__low low
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__vfs vfs
keywords__ 
time__1214670159000000 1214670159000000
changetime__1214726319000000 1214726319000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)|@therve removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun set owner to @therve|
|-|-|

`test_vfs.py` is missing a copyright statement and module docstring.  Please add those, then apply.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_3"></a>|@therve set status to `closed`|
|-|-|

(In [24154]) Remove useless shutil.rmtree twisted.vfs.test.test_vfs.OSVFSTest.tearDown,
which could cause some intermittent failures under Windows.

Author: therve
Reviewer: exarkun
Fixes [#3320](https://github.com/twisted/twisted/issues/3320)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
